### PR TITLE
2.10.0 Changelog

### PR DESCRIPTION
**Pull request by enrico-usai**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,41 +7,71 @@ CHANGELOG
...
+- Improve retrieval of instance type info by using `DescribeInstanceType` API.
+- Remove `custom_awsbatch_template_url` configuration parameter.
+- Upgrade `pip` to latest version in virtual environments.
+- Upgrade image used by CodeBuild environment when building container images for Batch clusters.
 
 **BUG FIXES**
 
-- Fix cfn_scheduler_slots don't update when updating compute instance type by ``pcluster update``. 
+- Retrieve the right number of compute instance slots when instance type is updated.
+- Include user tags in Compute Fleet resources and EBS volumes.
```
Comment (mauri-melato) [inline CHANGELOG.md:64]: compute nodes

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,41 +7,71 @@ CHANGELOG
...
+- Fix `pcluster status` output when head node is stopped.
+- `pcluster update`:
+  - Fix issue when tags are specified but not changed.
+  - Fix issue when the `cluster` section label changed.
+  - Fix issue when `shared_dir` and `ebs_settings` are both configured in the `cluster` section.
+- Fix pre/post install script arguments management when using double quotes.
+- Fix queue name validator to properly check for capital letters.
+- Fix `enable_efa` parameter validation for `queue` section, that was wrongly checking the value from `compute_resource`
```
Comment (mauri-melato) [inline CHANGELOG.md:72]: It needs to be reworded

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,41 +7,71 @@ CHANGELOG
...
-  for hosting S3 resources used by cluster.
+- FSx Lustre:
+  - Add possibility to configure Auto Import policy through the new `auto_import_policy` parameter.
+  - Add support to HDD storage type and the new `storage_type` and `drive_cache_type` configuration parameters.
+- Create a CloudWatch Dashboard for the cluster, named `<clustername>-<region>`, including head node EC2 metrics and 
+  cluster logs. It can be disabled by configuring the `enable` parameter in the `dashboard` section.
+- Add `-r/-region` arg to `pcluster configure` command. If this arg is provided, configuration will 
+  skip region selection.
+- Add `-r/-region` arg to`ssh` and `dcv connect` commands.
+- Add `cluster_resource_bucket` parameter under `cluster` section to allow the use of a user-provided S3 bucket.
```
Comment (mauri-melato) [inline CHANGELOG.md:23]: Add `cluster_resource_bucket` parameter under `cluster` section to allow the user to specify an existing S3 bucket.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,41 +7,71 @@ CHANGELOG
...
+  - Libfabric: `libfabric-1.11.1amazon1.1` (from libfabric-1.10.1amazon1.1)
+  - Open MPI: `openmpi40-aws-4.0.5` (from openmpi40-aws-4.0.3)
   - Unifies installer runtime options across x86 and aarch64
-  - Introduces ``-g/--enable-gdr`` switch to install packages with GPUDirect RDMA support
+  - Introduces `-g/--enable-gdr` switch to install packages with GPUDirect RDMA support
   - Updates to OMPI collectives decision file packaging, migrated from efa-config to efa-profile
   - Introduces CentOS 8 support
-- CentOS 6 is no longer supported.
-- Upgrade image used by CodeBuild environment when building container images for Batch clusters.
 - Enable queue resizing on update without requiring to stop the compute fleet. Stopping the compute fleet is only
```
Comment (mauri-melato) [inline CHANGELOG.md:49]: This is an enhancement

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,41 +7,71 @@ CHANGELOG
...
+- `createami`:
+  - Add validation step to fail when using a base AMI created by a different version of ParallelCluster.
+  - Add validation step for AMI creation process to fail if the selected OS and the base AMI OS are not consistent.
+  - Add `--post-install` parameter to use a post installation script when building an AMI.
+  - Add the possibility to use a ParallelCluster base AMI.
+- Add possibility to change tags when performing a `pcluster update`.
+- Add validator for EBS volume size, type and IOPS.
+- Add validators for `shared_dir` parameter when used in both `cluster` and `ebs` sections.
+- Add validator `cfn_scheduler_slots` key in the `extra_json` parameter.
+- Upgrade image used by CodeBuild environment when building container images for Batch clusters.
```
Comment (demartinofra) [inline CHANGELOG.md:33]: we could specify that we upgraded from `aws/codebuild/amazonlinux2-x86_64-standard:1.0` to `aws/codebuild/amazonlinux2-x86_64-standard:3.0`

Comment (mauri-melato) [inline CHANGELOG.md:33]: This is just a change

**Review by demartinofra — APPROVED** [review]

**Pull request merged**